test/tests/syncing.rs

```diff
@@ -0,0 +1,49 @@
+use rpc_test::test_config::TestConfig;
+use starknet::{providers::{
+    jsonrpc::{HttpTransport, JsonRpcClient},
+    Provider,
+}, core::types::SyncStatusType};
+use url::Url;
+
+#[tokio::test]
+async fn syncing() {
+    let config =
+        TestConfig::new("./secret.json").expect("'./secret.json' must contain correct node urls");
+
+    let deoxys = JsonRpcClient::new(HttpTransport::new(
+        Url::parse(&config.deoxys).expect("Error parsing deoxys url"),
+    ));
+
+    let alchemy = JsonRpcClient::new(HttpTransport::new(
+        Url::parse(&config.alchemy).expect("Error parsing reference node url"),
+    ));
+
+    let response_deoxys = deoxys
+        .syncing()
+        .await
+        .expect("Error while getting sync status from deoxys");
+
+    let response_reference = alchemy
+        .syncing()
+        .await
+        .expect("Error while getting sync status from reference node");
+
+    assert!(compare_sync_status(response_deoxys, response_reference));
+}
+
+fn compare_sync_status(
```

Comment:
Small detail could you put your util function on top of the main functions (im a 42 norm fan)

Reply:
Also this function is not needed. Please verify directly a with b in syncing fn to avoid human mistakes

Reply:
it is not possible to directly compare 'a' and 'b' because the start_block will not be the same depending on the node restarts

Reply:
Then you just avoid checking on the start block field? What will happen if we miss it on one side will the tests pass?